src/main/java/trello/entities/Card.java

```diff
@@ -38,18 +38,18 @@ public void setId(final String id) {
     }
 
     /**
-     * This method is used get name.
-     * @return a string with the name of card.
+     * This method is used get title.
+     * @return a string with the title of card.
      */
-    public String getName() {
-        return name;
+    public String getTitle() {
+        return title;
     }
 
     /**
-     * This method is used to set name.
-     * @param name defines of input string with the name.
+     * This method is used to set title.
+     * @param title defines of input string with the title.
```

Comment:
Please, add white space above this line.

Reply:
Solved